src/renderer/components/+extensions/extension-install.store.ts

```diff
@@ -209,13 +209,27 @@ export class ExtensionInstallationStateStore {
     return ExtensionInstallationStateStore.InstallingExtensions.size;
   }
 
+  /**
+   * The current number of extensions uninstalling
+   */
+   @computed static get uninstalling(): number {
```

Comment:
Please don't use any decorators with `static` (won't work in `mobx-6`)
See all migration pitfalls in https://github.com/lensapp/lens/pull/2718